src/Actions/ManagesContacts.php

```diff
@@ -42,6 +42,24 @@ public function findContact($email)
         return array_shift($contacts);
     }
 
+    /**
+     * Find contact by ID.
+     *
+     * @param int $id
+     *
+     * @return Contact|null
+     */
+    public function findContactById($id)
+    {
+        $contacts = $this->transformCollection(
+            $this->get('contacts', ['query' => ['id' => $id]]),
```

Comment:
FYI, this works - but I don't think it's a documented parameter.

Reply:
Ok, I believe you ;)